mlrun/feature_store/retrieval/job.py

```diff
@@ -174,18 +180,18 @@ def target_uri(self):
 import mlrun.feature_store.retrieval
 from mlrun.datastore.targets import get_target_driver
 def merge_handler(context, vector_uri, target, entity_rows=None, 
-                  timestamp_column=None, drop_columns=None, with_indexes=None, query=None, join_type='inner', 
-                  engine_args=None, order_by=None):
+                  entity_timestamp_column=None, drop_columns=None, with_indexes=None, query=None, join_type='inner', 
+                  engine_args=None, order_by=None, start_time=None, end_time=None, timestamp_for_filtering=None,):
     vector = context.get_store_resource(vector_uri)
     store_target = get_target_driver(target, vector)
-    entity_timestamp_column = timestamp_column or vector.spec.timestamp_field
     if entity_rows:
         entity_rows = entity_rows.as_df()
 
     context.logger.info(f"starting vector merge task to {vector.uri}")
     merger = mlrun.feature_store.retrieval.{{{engine}}}(vector, **(engine_args or {}))
     merger.start(entity_rows, entity_timestamp_column, store_target, drop_columns, with_indexes=with_indexes, 
-                 query=query, join_type=join_type, order_by=order_by)
+                 query=query, join_type=join_type, order_by=order_by, start_time=start_time, end_time=end_time,
+                 timestamp_for_filtering=timestamp_for_filtering,)
```

Comment:
This would be forbidden by the linter if it were code rather than a string containing code.
```suggestion
                 timestamp_for_filtering=timestamp_for_filtering)
```